src/Core/Riganti.Selenium.Core/BrowserWrapper.cs

```diff
@@ -89,7 +89,7 @@ public void SetCssSelector()
         /// <summary>
         /// Gives path of url of active browser tab.
         /// </summary>
-        public string CurrentUrlPath => new Uri(CurrentUrl).GetLeftPart(UriPartial.Path);
+        public string CurrentUrlPath => new Uri(CurrentUrl).GetComponents(UriComponents.PathAndQuery | UriComponents.Fragment & ~UriComponents.HostAndPort, UriFormat.UriEscaped);
```

Comment:
You can simplify the expression to

```suggestion
        public string CurrentUrlPath => new Uri(CurrentUrl).GetComponents(UriComponents.PathAndQuery | UriComponents.Fragment, UriFormat.UriEscaped);
```